android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/calendar/CalendarHelper.kt

```diff
@@ -0,0 +1,63 @@
+package jp.hacks.smartbread.ui.main.wake.calendar
+
+import android.content.Context
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jphacks/FK_1908/blob/3067a996df98dd226886069766038df5040996c7/android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/calendar/CalendarHelper.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>